packages/strapi-plugin-content-manager/admin/src/containers/App/reducer.js

```diff
@@ -17,6 +17,12 @@ const initialState = fromJS({
 
 function appReducer(state = initialState, action) {
   switch (action.type) {
+    case EMPTY_STORE:
+      return state
```

Comment:
```js
case EMPTY_STORE:
  return undefined;
```

I'm not sure if Immutable will allow this but based on [Dan Abramov's answer](https://stackoverflow.com/questions/35622588/how-to-reset-the-state-of-a-redux-store/35641992#35641992) it seems to be the right way.

Reply:
I'll go with :
```js
case EMPTY_STORE:
  return initialState;
```